tests/plugins.test.ts

```diff
@@ -1,6 +1,7 @@
 import { colord, getFormat, extend } from "../src/";
 import a11yPlugin from "../src/plugins/a11y";
 import cmykPlugin from "../src/plugins/cmyk";
+import harmoniesPlugin from "../src/plugins/harmonies";
```

Comment:
Could I also ask you to add the plugin to package.json? 🙏

Reply:
Sure, my bad :)